### ISSUE TITLE
Incorrect rendering in Bibliography section (DOC)

### ISSUE THREAD

**Issue by manuel489** [metanorma/metanorma-iec#66]
Numbering is still shown in non-auto-fetched references, and "Bibliography" title should be center-aligned.

DOC:
![capture2-doc](https://user-images.githubusercontent.com/33627611/84401751-a0a67700-abd1-11ea-8d8a-2043942f2ecd.PNG)

PDF:
![capture2-pdf](https://user-images.githubusercontent.com/33627611/84401774-a69c5800-abd1-11ea-9796-0374e8115533.PNG)

Thanks!

**Comment by opoudjis** [comment]
So, metanorma-iso applies numbering to all Bibliography entries, whether they have identifier codes or not. metanorma-iec does not.

And the IEC Word template confirms it:

````
It is not necessary to number the bibliography, but if it is numbered:
•	References shall be numbered consecutively in the order they appear in the text.
•	Apply the IEC style BIBLIOGRAPHY-numbered to the references. 
DO NOT use endnotes for creating a numbered bibliography.
````

In fact, so does the example Bibliography in IEC-template_v5_User-guide.

So whatever advice I was following in dropping the number for IEC was incorrect. @Intelligent2013 IEC says that numbering on bibliography is optional, but Word and PDF should be consistent, so request you add numbers to bibliography consistent with ISO.

**Comment by opoudjis** [comment]
Looking at https://github.com/metanorma/metanorma-iec/issues/62, I see that I had removed numbering from IEC to match PDF output, but I'd done so clumsily. In fact, Metanorma *expects* there to be numbered citations for non-standard references: author-date citations are not currently implemented. And there is no reason to treat unnumbered citations (PDF) as more correct than numbered citations (DOC) in IEC: both IEC and ISO/IEC DIR 2 guidance are clear that it is optional, and it will be far less disruptive if it numbering is used. So rescinding the change I had made then.
